kythe/typescript/testdata/getter_setter.ts

```diff
@@ -0,0 +1,65 @@
+// Tests the behavior of getter and setter entries.
+
+// Both getters and setters
+class A {
+  prop = 0;
+
+  //- @foo defines/binding GetFoo=VName("A.foo:getter", _, _, _, _)
+  //- GetFoo.node/kind function
+  //- @foo defines/binding PropFoo=VName("A.foo", _, _, _, _)
+  //- PropFoo.node/kind function
```

Comment:
The property itself doesn't seem like a `function` node.  The Kythe team recently discussed getters/setters in another context and thought that these might better be represented as `variable` nodes with a subkind of `implicit`.

We also discussed representing the relation between getters and setters explicitly through that `variable` node with `property/{writes,reads}` edges:

```
//- GetFoo property/reads PropFoo
//- SetFoo property/writes PropFoo
```

What do you think?

Reply:
I like that. I'll throw it together.

Reply:
No, it's not yet available.  I'll look into adding it to the schema soon.